src/crudOps.test.js

```diff
@@ -0,0 +1,43 @@
+/* eslint-disable */

```

Comment:
- [ ] Please make sure all the linter rules are enabled. These are here to help us write clean code.